packages/python/plotly/plotly/figure_factory/_ohlc.py

```diff
@@ -161,91 +161,9 @@ def create_ohlc(open, high, low, close, dates=None, direction="both", **kwargs):
     >>> from plotly.figure_factory import create_ohlc
     >>> from datetime import datetime
 
-    >>> import pandas.io.data as web
```

Comment:
here I removed all of them because the doc was broken and we have the trace anyway